404-commercial.html

```diff
@@ -22,11 +22,6 @@
     <script src="https://assets.openshift.net/content/html5shiv.js"></script>
     <script src="https://assets.openshift.net/content/respond.js"></script>
     <![endif]-->
-    <!-- IE8 requires jQuery and Bootstrap JS to load in head to prevent rendering bugs -->
```

Comment:
This comment explains why these 3 script tags should not be moved out of the `head` tag. If we're ok with dropping support for IE8 users, this change should be fine